src/Event/Events/EventFromUDB2.php

```diff
@@ -132,8 +133,8 @@ private function getCalendar(array $calendarAsArray): Calendar
 
         if ($calendarType === 'periods') {
```

Comment:
Not code you changed now, but can we put a todo comment here that once we upgrade to PHP8 we can change calenderType to be an Enum?